gunicorn/workers/gtornado.py

```diff
@@ -66,24 +66,25 @@ def heartbeat(self):
                         pass
                 self.server_alive = False
             else:
-                if not self.ioloop._callbacks:
+                if IOLOOP_PARAMETER_REMOVED:
+                    for callback in self.callbacks:
+                        callback.stop()
                     self.ioloop.stop()
-
-    def init_process(self):
-        # IOLoop cannot survive a fork or be shared across processes
-        # in any way. When multiple processes are being used, each process
-        # should create its own IOLoop. We should clear current IOLoop
-        # if exists before os.fork.
-        IOLoop.clear_current()
```

Comment:
So we don't need to call `IOLoop.clear_current()` anymore? Could you please check that removing this wouldn't break anything with pre-5.0 Tornado versions?

Reply:
This souldn't have been removed, good catch. Sorry.

Reply:
@benoitc @tilgovi should we support older Tornado versions or just the latest stable version?

Reply:
@bivald thanks for the clarification! @benoitc @tilgovi nevermind my question about supporting older Tornado versions :)